### PR TITLE
Add .eb files for snpEff v3.6 and snpEff v4.1d

### PR DESCRIPTION
**Pull request by npklein**
In versionsuffix, source_urls, and sources I'm not sure if the string formatting is PEP8 correct.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/s/snpEff/snpEff-v3.6-Java-1.7.0_80.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import time
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-v3.6-Java-1.7.0_80.eb]: oh no, please don't :)

Is this really necessary?

File: easybuild/easyconfigs/s/snpEff/snpEff-v3.6-Java-1.7.0_80.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+
+dependencies = [(java, javaver)]
+
+seconds_after_epoch = int(time.time())
+source_urls = [('http://downloads.sourceforge.net/project/%s?ts=%s&use_mirror=autoselect' % (name.lower(), seconds_after_epoch))]
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-v3.6-Java-1.7.0_80.eb]: use `SOURCEFORGE_SOURCE` template instead:

```python
source_urls = [SOURCEFORGE_SOURCE]
```

File: easybuild/easyconfigs/s/snpEff/snpEff-v3.6-Java-1.7.0_80.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+
+dependencies = [(java, javaver)]
+
+seconds_after_epoch = int(time.time())
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-v3.6-Java-1.7.0_80.eb]: drop this?

File: easybuild/easyconfigs/s/snpEff/snpEff-v4.1d-Java-1.7.0_80.eb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+
+dependencies = [(java, javaver)]
+
+seconds_after_epoch = int(time.time())
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-v4.1d-Java-1.7.0_80.eb]: unless there's a very good reason for this, please drop this

File: easybuild/easyconfigs/s/snpEff/snpEff-v4.1d-Java-1.7.0_80.eb [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import time
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-v4.1d-Java-1.7.0_80.eb]: remove pls

File: easybuild/easyconfigs/s/snpEff/snpEff-4.1d-Java-1.7.0_80.eb [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+description = """SnpEff is a variant annotation and effect prediction tool. 
+It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+
+dependencies = [(java, javaver)]
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-4.1d-Java-1.7.0_80.eb]: move this block down, below `source` please

File: easybuild/easyconfigs/s/snpEff/snpEff-3.6-Java-1.7.0_80.eb [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+description = """SnpEff is a variant annotation and effect prediction tool. 
+It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+
+dependencies = [(java, javaver)]
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-3.6-Java-1.7.0_80.eb]: move down below `sources` please

File: easybuild/easyconfigs/s/snpEff/snpEff-3.6-Java-1.7.0_80.eb [hunk 8]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'PackedBinary'
+
+name = 'snpEff'
+version = '3.6'
+
+homepage = 'http://sourceforge.net/projects/snpeff/'
+description = """SnpEff is a variant annotation and effect prediction tool. 
+It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/snpEff/snpEff-3.6-Java-1.7.0_80.eb]: 1 leading space helps readability (we should be doing it across all easyconfigs, yeah)

File: easybuild/easyconfigs/s/snpEff/snpEff-4.1d-Java-1.7.0_80.eb [hunk 9]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'PackedBinary'
+
+name = 'snpEff'
+version = '4.1d'
+
+homepage = 'http://sourceforge.net/projects/snpeff/'
+description = """SnpEff is a variant annotation and effect prediction tool. 
+It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/snpEff/snpEff-4.1d-Java-1.7.0_80.eb]: ditto about 1 leading space in descriptions... (this relates to how a docstring ends up to become modulefile help)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (2 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/33b53ed22dcd89425dcf for a full test report.

**Comment by boegel** [comment]
test failed because of missing Java tarball, do ignore :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/412254e7865da89da66d for a full test report.

**Comment by boegel** [comment]
Good to go, thanks @npklein, and @fgeorgatos for reviewing!

**Pull request merged**